adafruit_tfmini.py

```diff
@@ -0,0 +1,150 @@
+# The MIT License (MIT)
+#
+# Copyright (c) 2018 ladyada for Adafruit Industries
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in
+# all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+# THE SOFTWARE.
+"""
+`adafruit_tfmini`
+====================================================
+
+A CircuitPython/Python library for Benewake's TF mini distance sensor
+
+* Author(s): ladyada
+
+Implementation Notes
+--------------------
+
+**Hardware:**
+
+**Software and Dependencies:**
+
+* Adafruit CircuitPython firmware for the supported boards:
+  https://github.com/adafruit/circuitpython/releases
+
+"""
+
+import time
+import struct
```

Comment:
We still have `ustruct` in ESP8266, instead of `struct`. This is the usual approach:
```
try:
    import struct
except ImportError:
    import ustruct as struct
```

Reply:
Actually, let me verify this. I don't remember `ustruct` getting renamed...but I can't verify it right now.

Reply:
Ok, so, `ustruct` **was** renamed across the board in 3.x. I guess `ustruct` would only be needed for 2.x compatibility, and if using with MicroPython. Thanks to @dhalbert and @jerryneedell for the assist on verification!

If you aren't worried about either of those cases, let me know and I can close this CR.

Reply:
yah im ok with not supporting 2.0 :)